codex/rest/api.nim

```diff
@@ -664,6 +664,34 @@ proc initDebugApi(node: CodexNodeRef, conf: CodexConf, router: var RestRouter) =
         trace "Excepting processing request", exc = exc.msg
         return RestApiResponse.error(Http500)
 
+  when codex_enable_api_debug_delete:
+    type
+      DebugDeleteParams = object
+        blockIndices* {.serialize.}: seq[int]
+
+    router.rawApi(
+      MethodPost,
+      "/api/codex/v1/debug/delete/{cid}") do (cid: Cid) -> RestApiResponse:
```

Comment:
I would prefer using HTTP method `DELETE` on a `block` resource, so basically

```
DELETE /api/codex/v1/block/{cid}
```

Reply:
I think there is currently no way to get the cid of a block. All I have is the cid of the manifest and the number of blocks in it, so with this I can pick and index.

Reply:
`DELETE /api/codex/v1/block/{cid}` - this should delete the manifest
`DELETE /api/codex/v1/block/{cid}?block=1&block=50..100` - should delete the first block and `..` would delete a range.

Reply:
Actually that's a decent idea @dryajov to use query params. Maybe then just `/api/codex/v1/data/...` instead of `/api/codex/v1/block/...` to be consistent with current endpoints?